python/lsst/consdb/hinfo.py

```diff
@@ -442,14 +443,16 @@ def get_kafka_config() -> KafkaConfig:
 
 logger = setup_logging("consdb.hinfo")
 
-instrument = os.environ["INSTRUMENT"]
-logger.info(f"Instrument = {instrument}")
+instrument = ""
```

Comment:
The idea here is to have a more specific error than the KeyError that would be returned if the instrument is not defined?  I'm not sure that an "Unrecognized instrument: " (with no instrument name) error is much better.

Also, `instrument = os.environ.get("INSTRUMENT", "")` may be more idiomatic.